wallet/src/de/schildbach/wallet/ui/send/ConfirmTransactionDialog.kt

```diff
@@ -47,21 +53,56 @@ class ConfirmTransactionDialog : BaseBottomSheetDialogFragment() {
         private const val ARG_PAYEE_VERIFIED_BY = "arg_payee_verified_by"
 
         @JvmStatic
-        fun createDialog(address: String, amount: String, amountFiat: String, fiatSymbol: String, fee: String, total: String,
-                         payeeName: String? = null, payeeVerifiedBy: String? = null, buttonText: String? = null): DialogFragment {
+        fun createDialog(
+            activity: FragmentActivity,
+            address: String, amount: String, amountFiat: String, fiatSymbol: String, fee: String, total: String,
+                         payeeName: String? = null, payeeVerifiedBy: String? = null, buttonText: String? = null,
+        ) {
             val dialog = ConfirmTransactionDialog()
-            val bundle = Bundle()
-            bundle.putString(ARG_ADDRESS, address)
-            bundle.putString(ARG_AMOUNT, amount)
-            bundle.putString(ARG_AMOUNT_FIAT, amountFiat)
-            bundle.putString(ARG_FIAT_SYMBOL, fiatSymbol)
-            bundle.putString(ARG_FEE, fee)
-            bundle.putString(ARG_TOTAL, total)
-            bundle.putString(ARG_PAYEE_NAME, payeeName)
-            bundle.putString(ARG_PAYEE_VERIFIED_BY, payeeVerifiedBy)
-            bundle.putString(ARG_BUTTON_TEXT, buttonText)
-            dialog.arguments = bundle
-            return dialog
+            val bundle = setBundle(address, amount, amountFiat, fiatSymbol, fee, total, payeeName, payeeVerifiedBy, buttonText)
+            showDialog(dialog, bundle, activity)
```

Comment:
@ClaudeHangui it's not obvious that `createDialog` calls `show`. These should either be different methods like in `AdaptiveDialog`, or this method should be renamed